packages/core/src/App/Components/Layout/Header/toggle-notifications.jsx

```diff
@@ -23,28 +25,35 @@ const ToggleNotificationsDrawer = ({
         </div>
     );
 
+    if (isMobile) {
+        return (
+            <div
+                className={classNames('notifications-toggle', {
+                    'notifications-toggle--active': is_visible,
+                })}
+            >
+                {notifications_toggler_el}
+                <NotificationsDialog is_visible={is_visible} toggleDialog={toggleDialog} />
+            </div>
+        );
+    }
+
     return (
         <div
             className={classNames('notifications-toggle', {
                 'notifications-toggle--active': is_visible,
             })}
         >
-            <DesktopWrapper>
-                <Popover
-                    classNameBubble='notifications-toggle__tooltip'
-                    alignment='bottom'
-                    message={tooltip_message}
-                    should_disable_pointer_events={should_disable_pointer_events}
-                    zIndex={9999}
-                >
-                    {notifications_toggler_el}
-                </Popover>
-                <NotificationsDialog is_visible={is_visible} toggleDialog={toggleDialog} />
-            </DesktopWrapper>
-            <MobileWrapper>
+            <Popover
+                classNameBubble='notifications-toggle__tooltip'
+                alignment='bottom'
+                message={tooltip_message}
+                should_disable_pointer_events={should_disable_pointer_events}
+                zIndex={9999}
```

Comment:
nit: `zIndex='9999'`
This popover prop is a string (can be checked in packages/components/src/components/types/popover.types.ts)